embedchain/vectordb/elasticsearch_db.py

```diff
@@ -0,0 +1,57 @@
+import os
+from typing import Callable, Optional
+
+from elasticsearch import Elasticsearch
+from elasticsearch.helpers import bulk
+
+from embedchain.vectordb.base_vector_db import BaseVectorDB
+
+
+class EsDB(BaseVectorDB):
+    """
+    Elasticsearch as vector database
+        :param embedding_fn: Function to generate embedding vectors.
+        :param config: Optional. elastic search client

```

Comment:
I guess I'm not deep enough into elastic search to know why you don't call `client` `client` if that's what it is.

Reply:
removed client param